dev/unix/boot-install.sh

```diff
@@ -100,36 +118,37 @@ notion_get_openssl_version() {
   fi
 }
 
-NOTION_LATEST_VERSION=$(notion_get_latest_release)
+VOLTA_LATEST_VERSION=$(volta_get_latest_release)
 
-notion_info 'Checking' "for existing Notion installation"
-notion_check_existing_installation "$NOTION_LATEST_VERSION"
+volta_info 'Checking' "for existing Volta installation"
+volta_check_legacy_installation
+volta_check_existing_installation "$VOLTA_LATEST_VERSION"
 
 
 case $(uname) in
     Linux)
-        if [[ "$NOTION_LATEST_VERSION" == 0.1* ]]; then
-          NOTION_OS=linux
+        if [[ "$VOLTA_LATEST_VERSION" == 0.1* ]]; then
```

Comment:
I don't think this is needed here. This will never evaluate to true, since this rename is happening after 0.4

Reply:
Ah, interesting, yes. I'll pull it out.

Reply:
Fixed in 9276e5ba.